.circleci/config.yml

```diff
@@ -78,13 +79,18 @@ commands:
           command: |
             nvm install $NODE_VERSION
             nvm use $NODE_VERSION
-            nvm install-latest-npm
+      - run:
+          name: 'Install npm'
+          shell: bash.exe
+          command: (set -x; npm install -g npm@<< pipeline.parameters.npm-version >>)
       - run:
           name: 'Install dependent modules'
           shell: bash.exe
+          # Only run the npm install if the npm list command fails to find the module locally installed
+          # Note I picked a random module.
           command: |
             npm config set scripts-prepend-node-path true
-            npm install
+            npm list @heroku/functions-core || npm install
```

Comment:
this change results in only running npm install on windows it the node_modules weren't previously cached.

Reply:
the module isn't present if the cache isn't loaded which occurs when there is a change to package-lock.json